components/scripts/lib/build-scan-online.sh

```diff
@@ -44,17 +44,53 @@ is_build_scan_metadata_missing() {
   return 0
 }
 
-fetch_and_read_build_scan_data() {
+fetch_and_read_single_build_scan() {
```

Comment:
I am not very happy with these function names. We have:

- `fetch_and_read_single_build_scan`
- `parse_single_build_scan`
- `fetch_and_read_build_scans`
- `parse_build_scans`

Reply:
If it helps, the use case for calling each one is clearly defined. 

`fetch_and_read_single_build_scan` is only called to get data to recreate a Build Scan (CI/Local experiments).

`fetch_and_read_build_scans` is called by all experiments so that the summary can be displayed.

Reply:
Maybe something like this:

- fetch_single_build_scan
- parse_single_build_scan
- fetch_build_scans_and_build_time_metrics
- parse_build_scans_and_build_time_metrics

Reply:
I like it. Updated in 0cb9dfd1e466d66c398b0493e93f8a5418c3b8c5.

Reply:
LGTM